spark/common/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/raster/RasterBandAccessors.scala

```diff
@@ -45,6 +52,28 @@ case class RS_SummaryStats(inputExpressions: Seq[Expression]) extends InferredEx
   }
 }
 
+case class RS_Band(inputExpressions: Seq[Expression]) extends InferredExpression(RasterBandAccessors.getBand _) with SerdeAware with CodegenFallback {
```

Comment:
No need to add SerdeAware and CodgenFallback here as InferredExpression already has those.